### PR TITLE
Update output slurm exceptions and plotIterations

### PR DESCRIPTION
**Pull request by robinhasse**
I extended the removal of low memory slurm options from @orichters to a formulation that allows to filter the slurm options for various output scripts with a regex. This allows to only consider direct mode for `plotIterations` as this script is interactive and does not work with slurm. It finishes in a couple of seconds and is not regularly used so making this script slurm-compatible does not seem necessary to me.
I also updated `plotIterations` to make the aesthetic mapping interactive and set the defaults according to our current (and probably exclusive) application of monitoring the price targets implementation by @Renato-Rodrigues.

### CODE REVIEW

File: output.R
Comment (orichters) [inline output.R:150]: I like how you catch the length 1 case, but I wonder if for safety reasons, one should make sure that `direct` is always offered. So replace Line 148 with
```
slurm_options <- unique(c("direct", grep(slurmExceptions, slurm_options, value = TRUE))
```
This way, also the second comment would be fine…

**Review by orichters — CHANGES_REQUESTED** [review]
looks fine, apart from getting the `direct` option back for `reporting`

**Comment by robinhasse** [comment]
> looks fine, apart from getting the `direct` option back for `reporting`

You are right, that got lost. If you think, direct should never be removed, I adopt your proposal. Otherwise I could add it to the regex for `reporting` with a `or` connection.

**Comment by orichters** [comment]
> > looks fine, apart from getting the `direct` option back for `reporting`
> 
> You are right, that got lost. If you think, `direct` should never be removed, I adopt your proposal. Otherwise I could add it to the regex for `reporting` with an `or` connection.

Yes, I think I would always offer `direct`. At the moment at least, I don't see a reporting where you should not be able to run it locally.

File: output.R
Comment (robinhasse) [reply output.R:289]: `direct` is now always available following your suggestion.

File: output.R
Comment (robinhasse) [reply output.R:150]: `direct` is now always available following your suggestion.

**Comment by orichters** [comment]
Sorry, I did an error: The line I proposed lacks a `)` at the end…

**Review by orichters — APPROVED** [review]
Fine from my side regarding the slurm choice. Better wait for Pascal's approval concerning the rest before merging.

**Review by pfuehrlich-pik — APPROVED** [review]

**Pull request merged**
